src/plugins/dashboard/public/application/top_nav/show_share_modal.tsx

```diff
@@ -55,8 +58,8 @@ export function ShowShareModal({
         },
       },
     },
-    share: { toggleShareContextMenu },
     initializerContext: { kibanaVersion },
+    share: { toggleShareContextMenu },
```

Comment:
Was this rearranged to be alphabetical? 🤣

Reply:
(╯°□°)╯︵ ┻━┻

Sorry, it's automatic hahaha 😆